lib/workers/repository/update/branch/reuse.ts

```diff
@@ -62,7 +62,8 @@ export async function shouldReuseExistingBranch(
     (config.rebaseWhen === 'auto' &&
       (config.automerge || (await platform.getRepoForceRebase())))
   ) {
-    if (await isBranchBehindBase(branchName)) {
+    // TODO: fix types (#7154)
+    if (await isBranchBehindBase(branchName, baseBranch!)) {
```

Comment:
Fixed in a separate PR, right? To be merged first?

Reply:
No we change BranchCache in this PR so we will merge this first.